brianlong/www.validators.app
# 183769802 - use find_each on large ActiveRecord relations

#### What's this PR do?
PR adds find_each method for large ActiveRecord relations. Using it instead of each method.

#### How should this be manually tested?
- `rails test`

Changes impact on the following tasks, so please run them to make sure they work fully correct:
- `daemons/gather_rpc_mainnet_daemon.rb`
- `daemons/gather_vote_account_details_daemon.rb`
- perform in console: `ValidatorCheckActiveService.new.update_validator_activity`
- `script/validator_ips/set_active_ip_for_gossip_nodes.rb`
- `script/one_time_scripts/back_fill_commission_history.rb`
- `script/one_time_scripts/set_is_active_in_validator_ips.rb`

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183769802)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
